tests/test_ingv_readers.py

```diff
@@ -35,7 +35,7 @@ def test_cat_emrcmt(self):
         target_depth = []
         target_mw = []
         with open(filepath) as file_:
-            for targetline in file_.readlines():
+            for targetline in file_.readlines()[1:]:
```

Comment:
same as above

Reply:
About the multiple commits, they are merges from upstream. Let me know if there is a better way to handle that

Reply:
we can just squash all those commits when merging.